### ISSUE TITLE
Impossible to include some standard C++ header files

### ISSUE THREAD

**Issue by jfgo20** [Yakindu/statecharts#2892]
As discussed in [Google groups](https://groups.google.com/forum/#!topic/yakindu-user/5q6yGpXhfy8) it's for example impossible to include cmath or math.h in a project.

**Comment by tkutz** [comment]
@rherrmannr isn't this already fixed?

**Comment by jfgo20** [comment]
Not in release 3.5.10.

**Comment by rherrmannr** [comment]
Yes, you're right. I spend half a day on it. There are different reasons why we currently can't import/use all C/C++ headers, because of their complexity. 

We need to analyze the headers and put them into different categories for testing.
